ci/test_notebooks.sh

```diff
@@ -5,6 +5,8 @@ set -Eeuo pipefail
 
 . /opt/conda/etc/profile.d/conda.sh
 
+RAPIDS_VERSION="$(rapids-version)"
```

Comment:
Just picking a random place to comment, so the conversation can be threaded.

The `conda-python-notebooks` tests are failing like this:

```text
CUDARuntimeError: cudaErrorNoDevice: no CUDA-capable device is detected

During handling of the above exception, another exception occurred:

AttributeError                            Traceback (most recent call last)
File /tmp/dataset_from_disk_cudf-test.py:33
     31 import torch
     32 from rmm.allocators.torch import rmm_torch_allocator
---> 33 rmm.reinitialize(initial_pool_size=15e9)
     34 #Switch to async pool in case of memory issues due to fragmentation of the pool
     35 #rmm.mr.set_current_device_resource(rmm.mr.CudaAsyncMemoryResource(initial_pool_size=15e9))
     36 torch.cuda.memory.change_current_allocator(rmm_torch_allocator)

File /opt/conda/envs/test/lib/python3.10/site-packages/rmm/rmm.py:83, in reinitialize(pool_allocator, managed_memory, initial_pool_size, maximum_pool_size, devices, logging, log_file_name)
     80 for func, args, kwargs in reversed(_reinitialize_hooks):
     81     func(*args, **kwargs)
---> 83 mr._initialize(
     84     pool_allocator=pool_allocator,
     85     managed_memory=managed_memory,
     86     initial_pool_size=initial_pool_size,
     87     maximum_pool_size=maximum_pool_size,
     88     devices=devices,
     89     logging=logging,
     90     log_file_name=log_file_name,
     91 )

File memory_resource.pyx:956, in rmm.pylibrmm.memory_resource._initialize()

File memory_resource.pyx:995, in rmm.pylibrmm.memory_resource._initialize()

AttributeError: 'CUDARuntimeError' object has no attribute 'msg'
```

([build link](https://github.com/rapidsai/cugraph-gnn/actions/runs/11671232562/job/32499729390?pr=62))

Running this notebook: https://github.com/rapidsai/cugraph-gnn/blob/branch-24.12/python/cugraph-dgl/examples/dataset_from_disk_cudf.ipynb

@alexbarghi-nv does something about that need to be updated for newer versions of `rmm`?

Reply:
Yep, looks like that was the last thing we needed!

https://github.com/rapidsai/cugraph-gnn/actions/runs/11724718517/job/32660553648?pr=62

Thanks @tingyu66